jsonschema_gentypes/__init__.py

```diff
@@ -690,21 +701,22 @@ def get_name(
     name = normalize(name)
 
     prefix = "" if has_title else "_"
+    rand = str(random.randint(0, 4006)) if name != "Root" else ""
```

Comment:
Random names are not usable, we should keep a list of already known names and add a random number only if needed.